Draw an overflowed text only for left aligned wxGrid cells

thesiv opened the pull request.

thesiv committed:
Draw an overflowed text only for left aligned wxGrid cells

vadz reviewed (changes requested):
The logic is fine, but I don't like having code checking for `wxALIGN_LEFT` in 2 completely different places, it's not going to be obvious to find the other place to update if it ever changes in one of them.

I think we should add a helper function to `wxGridCellAttr` checking for this, e.g. `bool CanOverflow()` or something like this, that would check both the attribute and the alignment and use it in both places.

thesiv committed:
Add CanOverflow function to wxGridCellAttr

Add function to determine whether the cell will draw the overflowed text
to neighbour cells. Note that only left aligned cells currently can overflow.

thesiv committed:
fixup! Draw an overflowed text only for left aligned wxGrid cells

thesiv commented:
Fixed.
